Week3/mybusiness.sql

```diff
@@ -0,0 +1,189 @@
+INSERT INTO `employees` VALUES (1,'emre','1991-06-15','Netherland','12344566','earas446@gmail.com');
+/*!40000 ALTER TABLE `employees` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `project_info`
+--
+
+DROP TABLE IF EXISTS `project_info`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `project_info` (
+  `project_id` int(11) NOT NULL,
+  `department_id` int(11) NOT NULL,
+  KEY `project_id` (`project_id`),
+  KEY `department_id` (`department_id`),
+  CONSTRAINT `project_info_ibfk_1` FOREIGN KEY (`project_id`) REFERENCES `projects` (`project_id`),
+  CONSTRAINT `project_info_ibfk_2` FOREIGN KEY (`department_id`) REFERENCES `departments` (`dep_number`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `project_info`
+--
+
+LOCK TABLES `project_info` WRITE;
+/*!40000 ALTER TABLE `project_info` DISABLE KEYS */;
+/*!40000 ALTER TABLE `project_info` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `project_team`
+--
+
+DROP TABLE IF EXISTS `project_team`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `project_team` (
+  `employee_id` int(11) NOT NULL,
+  `project_id` int(11) NOT NULL,
+  KEY `employee_id` (`employee_id`),
+  KEY `project_id` (`project_id`),
+  CONSTRAINT `project_team_ibfk_1` FOREIGN KEY (`employee_id`) REFERENCES `employees` (`id`),
+  CONSTRAINT `project_team_ibfk_2` FOREIGN KEY (`project_id`) REFERENCES `projects` (`project_id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `project_team`
+--
+
+LOCK TABLES `project_team` WRITE;
+/*!40000 ALTER TABLE `project_team` DISABLE KEYS */;
+/*!40000 ALTER TABLE `project_team` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `projects`
+--
+
+DROP TABLE IF EXISTS `projects`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `projects` (
+  `project_id` int(11) NOT NULL AUTO_INCREMENT,
+  `project_name` varchar(50) NOT NULL,
+  `starting_date` date NOT NULL,
+  `status` varchar(50) NOT NULL,
+  PRIMARY KEY (`project_id`)
+) ENGINE=InnoDB AUTO_INCREMENT=2 DEFAULT CHARSET=utf8;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `projects`
+--
+
+LOCK TABLES `projects` WRITE;
+/*!40000 ALTER TABLE `projects` DISABLE KEYS */;
+INSERT INTO `projects` VALUES (1,'HYF','2018-04-15','in process');
+/*!40000 ALTER TABLE `projects` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `salaries`
+--
+
+DROP TABLE IF EXISTS `salaries`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `salaries` (
```

Comment:
This might be better as an attribute of the employees table, since it's probably one-to-one.